### PR TITLE
Merge from main to live for WAM updates

### PR DESCRIPTION
**Pull request by dbradish-microsoft**

### CODE REVIEW

File: docs-ref-conceptual/authenticate-azure-cli-interactively.md [hunk 1]
```diff
@@ -14,7 +14,30 @@ keywords: az login, authentication types, authentication methods, azure, cli log
 
 # Sign in interactively with Azure CLI 
 
-The Azure CLI's default authentication method for logins uses a web browser and access token to sign in.
+Interactive logins to Azure offer a more intuitive and flexible user experience. Interactive login with Azure CLI allows users to authenticate to Azure directly through the [az login](/cli/azure/reference-index#az-login) command, which is useful for ad-hoc management tasks and for environments that require manual sign-in, such as those with multi-factor authentication (MFA). This method simplifies access for script testing, learning, and on-the-fly management without needing to pre-configure service principals or other noninteractive authentication methods.
+
+## Sign in with Web Account Manager (WAM) on Windows
+
+As of Azure CLI version [2.61.0](./release-notes-azure-cli.md#may-21-2024), WAM is now the default authentication method on Windows. WAM is a Windows 10+ component that acts as an authentication broker. (An authentication broker is an application that runs on a user’s machine that manages the authentication handshakes and token maintenance for connected accounts.)
```
Comment (dcaro) [inline docs-ref-conceptual/authenticate-azure-cli-interactively.md:21]: Can we use "Web Account Manager" in one place so people know what WAM stands for?

File: docs-ref-conceptual/authenticate-azure-cli-interactively.md [hunk 2]
```diff
@@ -14,7 +14,30 @@ keywords: az login, authentication types, authentication methods, azure, cli log
...
+
+```azurecli-interactive
+az account clear
+az config set core.enable_broker_on_windows=false
+az login
+```
+
+## Sign in with a browser
+
+WAM is available on Windows 10 and later, and on Windows Server 2019 and later. On Mac, Linux, and earlier versions of Windows, the Azure CLI will continue to automatically default to a browser.
```
Comment (dcaro) [inline docs-ref-conceptual/authenticate-azure-cli-interactively.md:40]: Mentioning WAM here may be confusing since it is not browser based.

**Review by dcaro — COMMENTED** [review]
Please address the couple of comments

**Review by dcaro — APPROVED** [review]

File: docs-ref-conceptual/authenticate-azure-cli-interactively.md (hunk 1 above)
Comment (dbradish-microsoft) [reply docs-ref-conceptual/authenticate-azure-cli-interactively.md:21]: @dcar, "in one place" inside the paragraph in addition to the H2?  Sure.

**Comment by dbradish-microsoft** [comment]
Merging to `live` branch with approval from @dcaro.  Suggestions in this PR can be found in https://github.com/MicrosoftDocs/azure-docs-cli/pull/4319 which will publish to `live` later today or tomorrow, May 22.

**Pull request merged**